tests/s25Main/notificationSystem/TestNote1.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021 - 2021 Settlers Freaks (sf-team at siedler25.org)
+//
+// This file is part of Return To The Roots.
+//
+// Return To The Roots is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// Return To The Roots is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Return To The Roots. If not, see <http://www.gnu.org/licenses/>.
+
+#pragma once
+
+#include "notifications/notifications.h"
+
+struct TestNote1
+{
+    ENABLE_NOTIFICATION(TestNote1)
+
+    int value = 0;
```

Comment:
shouldn't we use something different for the values?

Reply:
This is only the default value so to avoid a ctor. Actual values are set via usage `TestNote1{42}`

Reply:
ah okay